PlanningBiblio/PlanningBiblio
# Move files

- Move calendar, position and statistics JS files to /public/js
- Remove unused class.modeles.php
- Fix misused of $this->config on statisticController (bug)
- Optimize calendar's display (width : 100%)

TEST PLAN : 
- Edit a calendar, check if its JS file is linked. Check if the appearance is alright. Columns should have the same width.
- Edit a position and check if its JS file is linked.
- Add "20h-21h" into Statistiques-Heures (administration / configuration / statistiques)
- Show statistics by agents, status, service, saturday. Change "Heures" value and submit, then click on "Charger les heures par défaut".
